src/platform/windows/angle/device.rs

```diff
@@ -267,10 +238,9 @@ impl Device {
         let d3d11_device = device as *mut ID3D11Device;
 
         unsafe {
-            (*d3d11_device).AddRef();
             Ok(Device {
                 egl_display: egl_display,
-                d3d11_device: ComPtr::from_raw(d3d11_device),
+                d3d11_device: (*d3d11_device).to_owned(),
```

Comment:
It would be better to call ID3D11Device's method so it will be more clear when reading this.